src/cowboy_swagger.erl

```diff
@@ -68,6 +69,12 @@
 -type metadata() :: trails:metadata(swagger_map()).
 -export_type([swagger_map/0, metadata/0]).
 
+-define(SWAGGER_2_0, swagger_2_0).
```

Comment:
We don't really like to use macros for these things. Can you just use the atoms everywhere and trust dialyzer? :)

Reply:
Done!